client/src/components/ColorButton.jsx

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+import { RadioButtonContainer, RadioButton, RadioButtonLabel } from '../styling.jsx';
+
+const ColorButton = (props) => {
+  return (
+    <RadioButtonContainer>
+      <RadioButton value={props.color.toLowerCase()} onClick={props.selectColor}/>
```

Comment:
Conversely, I think a space is usually inserted before the closing slash in single-tag elements like this:

`<RadioButton value={props.color.toLowerCase()} onClick={props.selectColor} />`

Reply:
Yes, that's exactly the rule I got the previous commit confused with :)